vendor/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v0.9.4
+ * Bump lower-bound SDK constraint to `2.14`.
```

Comment:
You really want this in the same PR?

Reply:
No, I want my tests to pass :see_no_evil: